src/content_cao.cpp

```diff
@@ -895,8 +895,11 @@ void GenericCAO::addToScene(ITextureSource *tsrc)
 	scene::ISceneNode *node = getSceneNode();
 	if (node && m_prop.nametag != "" && !m_is_local_player) {
 		// Add nametag
+		v3f pos;
+		pos.Y = m_prop.collisionbox.MaxEdge.Y + 0.1f;
```

Comment:
Probably needs to be higher, as before it was position + 1.1, which is 2.1 above foot position, the collisionbox top is at 1.77 above foot position, so add 0.3f?